### PR TITLE
Playground: accept batched messages

### PR DESCRIPTION
**Pull request by molly-moen**
[This PR](https://github.com/code-dot-org/javabuilder/pull/133) updates playground to send messages in batches. Update playground to accept these new batched messages and iterate through the updates one by one. 

## Links

- jira ticket: [CSA-842](https://codedotorg.atlassian.net/browse/CSA-842)

## Testing story
Tested locally

## Deployment strategy
This change should be deployed before the javabuilder change to prevent disruptions in the playground.

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/javalab/Playground.js [hunk 1]
```diff
@@ -127,6 +131,12 @@ export default class Playground {
     this.setIsProgramRunning(false);
   }
 
+  handleBatchUpdate(details) {
+    details.updates.forEach(data => {
+      this.handleSignal(data);
+    });
+  }
```
Comment (sanchitmalhotra126) [inline apps/src/javalab/Playground.js:138]: possible to add a test case for this?

**Review by sanchitmalhotra126 — COMMENTED** [review]
Nice, good call keeping this backwards-compatible! Would be nice to add a test case for the new signal, otherwise LGTM

File: apps/src/javalab/Playground.js (hunk 1 above)
Comment (molly-moen) [reply apps/src/javalab/Playground.js:138]: this method is pretty simple and essentially wraps `handleSignal`, what parts of it do you think are worth testing?

Comment (sanchitmalhotra126) [reply apps/src/javalab/Playground.js:138]: Hmm yeah, I was thinking a simple test to ensure the UPDATES signal properly calls each individual update, but I guess that's effectively just verifying that `forEach` works, so may not be necessary.

**Review by sanchitmalhotra126 — APPROVED** [review]

**Pull request merged**